tests/benchmarks/cpu/intersect_surfaces.cpp

```diff
@@ -1,71 +1,50 @@
 /** Detray library, part of the ACTS project (R&D line)
  *
- * (c) 2020-2022 CERN for the benefit of the ACTS project
+ * (c) 2020-2023 CERN for the benefit of the ACTS project
  *
  * Mozilla Public License Version 2.0
  */
 
-// Project include(s)
+// Detray core include(s).
+#include "detray/definitions/containers.hpp"
 #include "detray/definitions/indexing.hpp"
 #include "detray/intersection/concentric_cylinder_intersector.hpp"
-#include "detray/intersection/cylinder_intersector.hpp"
 #include "detray/intersection/detail/trajectories.hpp"
 #include "detray/masks/masks.hpp"
-#include "detray/materials/material_slab.hpp"
+
+// Detray utility include(s).
 #include "detray/simulation/event_generator/track_generators.hpp"
-#include "tests/common/tools/test_surfaces.hpp"
+
+// Detray test include(s).
+#include "detray/test/planes_along_direction.hpp"
 
 // Google Benchmark include(s)
 #include <benchmark/benchmark.h>
 
-// System include(s)
-#include <fstream>
-
-using namespace detray;
-
-#ifdef DETRAY_BENCHMARKS_REP
-int gbench_repetitions = DETRAY_BENCHMARKS_REP;
-#else
-int gbench_repetitions = 0;
-#endif
-
-enum mask_ids : unsigned int {
-    e_rectangle2 = 0,
-    e_cylinder2 = 1,
-    e_conc_cylinder3 = 2,
-};
-
-enum material_ids : unsigned int {
-    e_slab = 0,
-};
-
-using mask_link_t = dtyped_index<mask_ids, dindex>;
-using material_link_t = dtyped_index<material_ids, dindex>;
-
-using plane_surface = surface<mask_link_t, material_link_t, transform3>;
+static const unsigned int theta_steps = 1000u;
+static const unsigned int phi_steps = 1000u;
 
-unsigned int theta_steps = 1000u;
-unsigned int phi_steps = 1000u;
-
-dvector<scalar> dists = {1.f, 2.f, 3.f, 4.f, 5.f, 6.f, 7.f, 8.f, 9.f, 10.f};
+static const detray::dvector<detray::scalar> dists = {1.f, 2.f, 3.f, 4.f, 5.f,
+                                                      6.f, 7.f, 8.f, 9.f, 10.f};
 
 // This test runs intersection with all surfaces of the TrackML detector
 static void BM_INTERSECT_PLANES(benchmark::State &state) {
 
     unsigned int sfhit = 0u;
     unsigned int sfmiss = 0u;
 
-    auto planes = planes_along_direction(
-        dists, vector::normalize(vector3{1.f, 1.f, 1.f}));
-    constexpr mask<rectangle2D<>> rect{0u, 10.f, 20.f};
-    point3 ori = {0.f, 0.f, 0.f};
+    auto planes = detray::test::planes_along_direction(
+        dists, detray::vector::normalize(detray::test::vector3{1.f, 1.f, 1.f}));
+    constexpr detray::mask<detray::rectangle2D<>> rect{0u, 10.f, 20.f};
+    detray::test::point3 ori = {0.f, 0.f, 0.f};
```

Comment:
Let me highlight the lines right after the one that I opened the https://github.com/acts-project/detray/pull/437/files#r1159395254 discussion on. (I should've tagged all these lines with that comment.)

I believe it does make things quite a bit clearer how this looks compared to how it used to. As I, as a "new reader" of the code, spent some time finding what `vector3` and `point3` really meant in this context. :thinking: Calling them explicitly `detray::test::vector3` and `detray::test::point3` improves readability in my mind greatly... :thinking: